docs/writing.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Writing Documents
+nav_order: 4
+---
+
+The MarkLogic MuleSoft connector supports writing batches of documents of any type to MarkLogic in a scalable, 
+performant manner via Mule's [support for batch processing](https://docs.mulesoft.com/mule-runtime/latest/batch-processing-concept). 
+While using Mule's batch processing support is not required, it is recommended regardless of the amount of data that
+your flow will write to MarkLogic.
+
+## Table of contents
+{: .no_toc .text-delta }
+
+- TOC
+{:toc}
+
+## Configuring the operation
+
+Once your flow has a source of data, you can add support for writing documents to MarkLogic. The following steps 
+describe how to do so within Mule's batch processing feature (note that these steps are describing one approach; 
+please see the Mule documentation for complete information in its batch processing feature):
+
+1. In the Mule Palette, click on the "Core" module and drag a "Batch Job" into your flow. 
+2. From the "Core" module, drag a "Batch Aggregator" into the "Process Records" area of the "Batch Job" in your flow.
```

Comment:
"Batch Step" might be more suitable here. "Process Records" label is there as well, but since we are adding "Write Documents" to aggregator of the Batch Step we can say - "... drag a "Batch Aggregator" in the Aggregator field of Batch Step in the Batch Job in your flow."